perfdash/metrics-downloader.go

```diff
@@ -62,6 +57,7 @@ func NewDownloader(opt *DownloaderOptions, bkt MetricsBucket) *Downloader {
 // TODO(random-liu): Only download and update new data each time.
 func (g *Downloader) getData() (JobToCategoryData, error) {
 	configPaths := make([]string, len(g.Options.ConfigPaths))
+	fmt.Println(len(configPaths))
```

Comment:
Let's not use fmt - use klog for consistency.

Reply:
I was using this line for debugging and forgot to remove it - will do so now